docs/installation/sources/sources_windows.rst

```diff
@@ -0,0 +1,376 @@
+    command. For more information on the specific syntax, please refer to the
+    `CMake specific arguments <https://colcon.readthedocs.io/en/released/reference/verb/build.html#cmake-specific-arguments>`_
+    page of the colcon_ manual.
+
+.. _run_app_colcon_sw:
+
+Run an application
+^^^^^^^^^^^^^^^^^^
+
+When running an instance of an application using *eProsima Fast DDS*, the colcon overlay built in the
+dedicated :code:`Fast-DDS` directory must be sourced.
+There are two possibilities:
+
+* Every time the :code:`Fast-DDS` directory is opened in a new shell, prepare the environment locally by typing the
+  command:
+
+  .. code-block:: bash
+
+      setup.bat
+
+* Add the sourcing of the colcon overlay permanently, by opening the
+  *Edit the system environment variables* control panel, and adding :code:`~/Fast-DDS/install/setup.bat`
+  to the :code:`PATH`.
+
+
+.. _cmake_installation_windows:
+
+CMake installation
+-------------------
+
+This section explains how to compile *eProsima Fast DDS* with CMake_, either :ref:`locally <local_installation_sw>` or
+:ref:`globally <global_installation_sw>`.
+
+.. _local_installation_sw:
+
+Local installation
+^^^^^^^^^^^^^^^^^^
+
+#. Open a command prompt, and create a :code:`Fast-DDS` directory where to download and build *eProsima Fast DDS* and
+   its dependencies:
+
+   .. code-block:: bash
+
+       mkdir ~/Fast-DDS
+
+#. Clone the following dependencies and compile them using CMake_.
+
+   * `Foonathan memory <https://github.com/foonathan/memory>`_
+
+     .. code-block:: bash
+
+         cd ~/Fast-DDS
+         git clone https://github.com/eProsima/foonathan_memory_vendor.git
+         cd foonathan_memory_vendor
+         mkdir build && cd build
+         cmake .. -DCMAKE_INSTALL_PREFIX=~/Fast-DDS/install -DBUILD_SHARED_LIBS=ON
+         cmake --build . --target install
+
+   * `Fast CDR <https://github.com/eProsima/Fast-CDR.git>`_
+
+     .. code-block:: bash
+
+         cd ~/Fast-DDS
+         git clone https://github.com/eProsima/Fast-CDR.git
+         cd Fast-CDR
+         mkdir build && cd build
+         cmake .. -DCMAKE_INSTALL_PREFIX=~/Fast-DDS/install
+         cmake --build . --target install
+
+#. Once all dependencies are installed, install *eProsima Fast DDS*:
+
+   .. code-block:: bash
+
+       cd ~/Fast-DDS
+       git clone https://github.com/eProsima/Fast-RTPS.git
+       cd Fast-RTPS
+       mkdir build && cd build
+       cmake ..  -DCMAKE_INSTALL_PREFIX=~/Fast-DDS/install -DCMAKE_PREFIX_PATH=~/Fast-DDS/install
+       cmake --build . --target install
+
+.. _global_installation_sw:
+
+Global installation
+^^^^^^^^^^^^^^^^^^^
+
+To install *eProsima Fast DDS* system-wide instead of locally, remove all the flags that
+appear in the configuration steps of :code:`Fast-CDR` and :code:`Fast-RTPS`, and change the first in the
+configuration step of :code:`foonathan_memory_vendor` to the following:
+
+.. code-block:: bash
+
+    -DCMAKE_INSTALL_PREFIX=??
```

Comment:
You don't need the `DCMAKE_INSTALL_PREFIX` flag, it should use the default, which in Windows is `C:\Program Files\${PACKAGE_NAME}`, in this case:
- Foonathan memory -> `C:\Program Files\memory`
- Fast CDR -> `C:\Program Files\fastcdr`
- Fast DDS -> `C:\Program Files\fastrtps`

Furthermore, you don't need `DCMAKE_PREFIX_PATH`, since CMake will look for the packages in the above directories.

Reply:
remove the install_prefix and the prefix_path. shared_on stays.